docs/usage.rst

```diff
@@ -142,6 +142,38 @@ separate functions based on where the model is being run.
 .. autofunction:: fv3config.run_native
 .. autofunction:: fv3config.run_docker
 
+Submitting a Kubernetes job
+---------------------------
+
+A python interface is provided for submitting `fv3run` jobs to Kubernetes. Here's
```

Comment:
I don't think that would be appropriate to put here. Users can use kubectl or possibly the web console to get this info. I will however have the example use `gcp-key` as the secret name.

Reply:
Why not? Currently we are the only users of this package. I don't see why we can't add some information that will make our lives easier.

Reply:
I added the note to the `fv3net` readme that our vcm key name is `gcp_key`. I think that is the "right" place to put internal onboarding documentation as to how our workflows work and how our credentials are set up. That coupled with the example value being `gcp_key` should be enough to tell people what key to use.

On top of that, `kubectl get secrets` immediately says we have a gcp_key secret.

Even if we're the only people using this package right now, the idea is to make it so that's not the case. We have some steps to get there still but I'd like to step forward. Personally I'd be afraid to use a package that goes as far as to say what their internal credentials secret is called in the package documentation.